custom_components/tuya_local/devices/pool_thermometer.yaml

```diff
@@ -0,0 +1,36 @@
+name: Raddy PT-5 pool thermometer
+products:
+  - id: qirx90k6hg4qzpbz
+    model: EM3406B-TY
+entities:
+  - entity: sensor
+    name: indoor temperature
```

Comment:
Is it really indoor? Maybe "Ambient temperature" or "Air temperature" would be better to handle both indoor and outdoor installations.

Entity names should have the first letter of the first word capitalised.